zerver/lib/streams.py

```diff
@@ -140,6 +139,25 @@ def send_stream_creation_event(
     send_event_on_commit(realm, event, user_ids)
 
 
+def get_stream_permission_default_group(setting_name: str, realm: Realm) -> UserGroup:
+    setting_default_name = Stream.stream_permission_group_settings[setting_name].default_group_name
+    return NamedUserGroup.objects.get(
+        name=setting_default_name,
+        realm=realm,
+        is_system_group=True,
+    )
+
+
+def get_default_group_setting_values(realm: Realm) -> dict[str, UserGroup]:
```

Comment:
We might want to rename this to end with `for_channel` or something; currently you have to check where it is imported from to get that context when reading code for it.

Reply:
Changed the name at #32538